api/app.js

```diff
@@ -123,6 +125,12 @@ if (cluster.isMaster){
                   in: 'header',
                   name: 'Authorization',
                   description: "",
+              },
+              Basic: {
```

Comment:
This is a good spot to add this, nice thinking. Could we also add it to the admin guide under the webhook section?

Reply:
Added it to the admin guide.